2020/day2/solution.py

```diff
@@ -0,0 +1,28 @@
+import re
+password_re = re.compile(r"(\d+)-(\d+) (\w): (\w+)")
+
+def parsePassword(line):
+    match = password_re.match(line)
+    return (int(match.group(1)), int(match.group(2)), match.group(3), match.group(4))
+
+def isValidPartOne(minCount, maxCount, character, password):
+    count = 0
+    count = sum(1 if x == character else 0 for x in password)
+    return count >= minCount and count <= maxCount
```

Comment:
```python
minCount <= count <= maxCount
```

Also - `snake_case` not `camelCase` 😄

Reply:
stupid c# conventions :)

I didn't know you  could chain inequality that way